repository/kafka/src/test/java/com/expediagroup/streamplatform/streamregistry/repository/kafka/SampleModel.java

```diff
@@ -55,6 +61,10 @@ static Specification specification() {
     specification.setTags(Collections.singletonList(new Tag("name", "value")));
     specification.setType("type");
     specification.setConfiguration(mapper.createObjectNode());
+    specification.setSecurity(java.util.stream.Stream.of(
```

Comment:
Kotlin would be nice here :'(

guava might help with the verbosity here

Reply:
yes, guava reduced the complexity, I am replacing all the repetitions.